src/libraries/System.Formats.Tar/tests/TarFile/TarFile.ExtractToDirectoryAsync.Stream.Tests.cs

```diff
@@ -110,6 +110,29 @@ public async Task ExtractEntry_DockerImageTarWithFileTypeInDirectoriesInMode_Suc
             }
         }
 
+        [Fact]
+        public async Task ExtractEntry_PodmanImageTarWithRelativeSymlinksPointingInExtractDirectory_SuccessfullyExtracts_Async()
```

Comment:
This has nothing to do with `podman`, so I would not include that in the test name.

Reply:
The other test before this one has nothing to do with docker either. Name makes it just easier to spot from logs.

Reply:
We can even build this file as part of a test, it's a valid one. (Unlike the one created by `docker`).

Reply:
`Podman` in this name is just noise imo.